bigbluebutton-html5/imports/api/common/server/helpers.js

```diff
@@ -23,6 +23,12 @@ export function appendMessageHeader(eventName, messageObj) {
 
 export function clearCollections() {
   console.log('in function clearCollections');
+
+  //This is to prevent collection clearing in development environment
```

Comment:
Would be helpfull to point out the current bug in meteor where the server refresh on client-side changes.